src/index.js

```diff
@@ -109,6 +110,14 @@ function App() {
                   }
                 />
 
+                <SecureRoute
+                  exact
+                  path="/leaderboard"
+                  component={() =>
+                    Authorization([Roles[0], Roles[1], Roles[2]], Leaderboard)
```

Comment:
Its not super clear here, but it looks like we are giving YDPs access to the leaderboard route. Is that what we want?

Reply:
Fixed this in most recent push,